.github/workflows/docker-image.yml

```diff
@@ -62,13 +62,28 @@ jobs:
           password: ${{ secrets.GITHUB_TOKEN }}
 
         # https://github.com/docker/build-push-action
-      - name: Build and push Docker image
+      - name: Build and push x86 Docker image
         uses: docker/build-push-action@3b5e8027fcad23fda98b2e3ac259d8d67585f671 #v4.0.0
         with:
-          file: .docker/Dockerfile
+          file: .docker/Dockerfile_x86
           push: true
-          platforms: linux/amd64,linux/arm64
+          platforms: linux/amd64
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
-          build-args: VERSION=${{ steps.get_version.outputs.version_build }}
+          build-args: |
+            VERSION=${{ steps.get_version.outputs.version_build }}
+            ARCH_BASE=eclipse-temurin:11-alpine
+          outputs: type=image
+
+      - name: Build and push ARM Docker image
+        uses: docker/build-push-action@3b5e8027fcad23fda98b2e3ac259d8d67585f671 #v4.0.0
+        with:
+          file: .docker/Dockerfile_ARM
```

Comment:
```suggestion
          file: .docker/Dockerfile
```

Reply:
The error you're getting is caused by `Docker` using the wrong context. To build the container you should point to the `Dockerfile` and set `.` as the context:

```
docker build -f .docker/Dockerfile -t shacl-test:latest --build-arg ARCH_BASE=eclipse-temurin:11-alpine --build-arg VERSION=v1.4.2 .
```

You also reminded me to update the `README` with the additional argument.

Reply:
Great, thank you! We should indeed have this command in the readme, it's quite tricky